### PR TITLE
Adjust textmate tag snippets

### PR DESCRIPTION
**Pull request by anthmatic**
This PR makes 3 changes:
- Adds `JinjavaTextMateSnippet` for some tags
- Filters out `hidden` code snippets
- Adjusts the output for [generated snippets](https://github.com/HubSpot/jinjava/pull/984/files#diff-1f25f0595c192fdbf51cc843e34c323ebbd0d462a0d4ca9a5f407d1a0f701f6bR357)

### CODE REVIEW

File: src/main/java/com/hubspot/jinjava/lib/tag/BlockTag.java [hunk 1]
```diff
@@ -15,10 +15,7 @@
  **********************************************************************/
 package com.hubspot.jinjava.lib.tag;
 
-import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
-import com.hubspot.jinjava.doc.annotations.JinjavaHasCodeBody;
-import com.hubspot.jinjava.doc.annotations.JinjavaParam;
-import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.doc.annotations.*;
```
Comment (jboulter) [inline src/main/java/com/hubspot/jinjava/lib/tag/BlockTag.java:18]: our style is to not use wildcard imports

File: src/main/java/com/hubspot/jinjava/lib/tag/IfTag.java [hunk 2]
```diff
@@ -15,10 +15,7 @@
  **********************************************************************/
 package com.hubspot.jinjava.lib.tag;
 
-import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
-import com.hubspot.jinjava.doc.annotations.JinjavaHasCodeBody;
-import com.hubspot.jinjava.doc.annotations.JinjavaParam;
-import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.doc.annotations.*;
```
Comment (jboulter) [inline src/main/java/com/hubspot/jinjava/lib/tag/IfTag.java:18]: wildcard import

File: src/main/java/com/hubspot/jinjava/lib/tag/MacroTag.java [hunk 3]
```diff
@@ -3,10 +3,7 @@
 import com.google.common.base.Splitter;
 import com.google.common.base.Strings;
 import com.google.common.collect.Lists;
-import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
-import com.hubspot.jinjava.doc.annotations.JinjavaHasCodeBody;
-import com.hubspot.jinjava.doc.annotations.JinjavaParam;
-import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.doc.annotations.*;
```
Comment (jboulter) [inline src/main/java/com/hubspot/jinjava/lib/tag/MacroTag.java:6]: wildcard import

File: src/main/java/com/hubspot/jinjava/lib/tag/UnlessTag.java [hunk 4]
```diff
@@ -1,9 +1,6 @@
 package com.hubspot.jinjava.lib.tag;
 
-import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
-import com.hubspot.jinjava.doc.annotations.JinjavaHasCodeBody;
-import com.hubspot.jinjava.doc.annotations.JinjavaParam;
-import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.doc.annotations.*;
```
Comment (jboulter) [inline src/main/java/com/hubspot/jinjava/lib/tag/UnlessTag.java:3]: wildcard input

**Review by jboulter — CHANGES_REQUESTED** [review]

File: src/main/java/com/hubspot/jinjava/lib/tag/BlockTag.java (hunk 1 above)
Comment (anthmatic) [reply src/main/java/com/hubspot/jinjava/lib/tag/BlockTag.java]: Fixed that, thanks for pointing that out

**Review by AlpriElse — APPROVED** [review]

**Pull request merged**